### ISSUE TITLE
Fatal error parsing malformed XML

### ISSUE THREAD

**Issue by meuxx** [kint-php/kint#247]
Hi,

I'm using Kint 2.1.1

Trying to dump a wrong XML like `Kint::dump('<?xml');` will produce the following fatal error 
```
Error thrown with message "Call to a member function getName() on boolean"  in ../vendor/kint-php/kint/src/Parser/Xml.php:75
```

This is because the errors parsing the XML are not being handled correctly:
In the line Xml.php:59  `$xml = simplexml_load_string($var);` will return `false` in case of error and not an exception.

Thank you!

**Comment by jnvsor** [comment]
Nice catch, I'll get to fixing that now

**Comment by jnvsor** [comment]
Fixed in 459fcdc1311743cea64a9215faa4eaaf32e81cb2
